api/openapi/model-registry.yaml

```diff
@@ -1661,7 +1661,7 @@ components:
       explode: true
       examples:
         orderBy:
-          value: Id
+          value: ID
```

Comment:
I'm not sure I follow this modification,
allegedly the current value was consistent with the enumeration here:

https://github.com/kubeflow/model-registry/blob/327e865819c865b9849ab89bb53730156e90675d/api/openapi/model-registry.yaml#L1351

Reply:
completely missed it! thanks. updated now.